### PR TITLE
chore: minecraft 1.17

### PR DESCRIPTION
**Pull request by marcus8448**
Drops cotton resources.
Includes @Littledog1229's loot tables and recipes
Marked as draft because REI is not implemented yet.

### CODE REVIEW

**Comment by craftablescience** [comment]
I switched priority to high, because if it's not on this, where are we going to use it? This affects the entire project as well, it's a pretty important PR

File: build.gradle.kts [hunk 1]
```diff
@@ -214,51 +253,52 @@ tasks.processResources {
...
 tasks.jar {
     from("LICENSE")
     manifest {
-        attributes(mapOf(
+        attributes(
             "Implementation-Title"     to modName,
             "Implementation-Version"   to project.version,
             "Implementation-Vendor"    to "Team Galacticraft",
             "Implementation-Timestamp" to DateTimeFormatter.ISO_DATE_TIME,
             "Maven-Artifact"           to "$modGroup:$modName:$project.version"
```
Comment (JoeZwet) [inline build.gradle.kts:273]:
```suggestion
            "Maven-Artifact"           to "$modGroup:$modName:${project.version}"
```
missing curly brackets for more than one word

**Review by JoeZwet — CHANGES_REQUESTED** [review]

File: src/main/java/dev/galacticraft/mod/GalacticraftMixinPlugin.java [hunk 2]
```diff
@@ -48,6 +48,9 @@ public boolean shouldApplyMixin(String targetClassName, String mixinClassName) {
         if (!FabricLoader.getInstance().isDevelopmentEnvironment()) {
             return !mixinClassName.equals("dev.galacticraft.mod.mixin.StructurePoolGeneratorMixin");
         }
+        if (mixinClassName.equals("dev.galacticraft.mod.mixin.client.WorldRendererOverworldMixin")) {
```
[the remark this answers is not in the document]
Comment (marcus8448) [reply src/main/java/dev/galacticraft/mod/GalacticraftMixinPlugin.java]: Exported to `Constant` in 899074c501f574aff52e775cd43ecfcd4756202b

File: build.gradle.kts (hunk 1 above)
Comment (marcus8448) [reply build.gradle.kts]: Fixed in f7fd2d664718bf7ee5f8d0c19c7f964e71fbad8d

**Review by JoeZwet — APPROVED** [review]

**Pull request merged**